contrib/packaging/k8s/daemon-set.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: cilium-net-controller
+spec:
+  template:
+    metadata:
+      labels:
+        name: cilium-net-controller
+    spec:
+      # nodeSelector:
+      #   with-network-plugin: cilium
+      containers:
+      - image: cilium/cilium:cilium-ubuntu-16-04
```

Comment:
Yes, it's the version pointed to the master branch plus it's the base image we are using to build cilium's docker image. We don't have an "official" release number.